Winch/Serialization/Item/HarvestableItemDataConverter.cs

```diff
@@ -2,7 +2,7 @@
 
 // ReSharper disable HeapView.BoxingAllocation
 
-namespace Winch.Serialization;
+namespace Winch.Serialization.Item;
```

Comment:
pre-emptive change since I will be adding POI category (which has more subcategories)